spec/podfile/dsl_spec.rb

```diff
@@ -44,6 +44,33 @@ module Pod
         target.pod_whitelisted_for_configuration?('PonyDebugger', 'Debug').should.be.false
       end
 
+      it 'allows specifying multiple additional subspecs' do
+        podfile = Podfile.new do
+          pod 'RestKit', '~> 0.24.0', :subspecs => %w(CoreData Networking), :configurations => %w(Release)
```

Comment:
With this implementation, this example shows that the root spec is also included.

```ruby
pod 'RestKit', :subspecs => %w(CoreData Networking)
```

Expands into:

- RestKit
- RestKit/CoreData
- RestKit/Networking

I'm not sure if it makes sense to include the root spec when using `:subspecs` since by default, the root spec *should* make the full library available unless their is alternative incompatible subspecs.

This means that this actually have no benefit for pods such as ARAnalytics since:

```ruby
pod 'ARAnaytics', :subspecs => ['Crashlytics', 'Mixpanel']
```

Would effectively do nothing since all the default_subspecs from ARAnalytics are included (which is all of them if i'm not mistaken).

I've created a PR #236 to change it's behaviour.